### DISCUSSION TITLE
Localisation - contributors list

### DISCUSSION THREAD

**Discussion by originalfoo** [CitiesSkylinesMods/TMPE#1250] [General]
Been chipping away at release notes for 11.6 and decided to build a full list of people who've participated in localisation (any interaction on crowdin).... 

@krzychu124 do you want this for the workshop page?

Adam Janík (jmenujuseadam), AduitSSH, Aimarekin, Alan Willian Duarte (nipodemos13), Alex (AlexofCA), Alexander Maximilian Hečko (bloxelstudiofficial), alvaroer, Anton Leibold (Cirez), aubergine18, BlackScout (BS_BlackScout), CaptainKlums, Chamëleon (chameleon-tbn), charlco, Chindanai B. (ItsMeChindanai), cianecollazzo (azzo94), Dávid Sz. (sz.david9), Den Olk (revolter00), DNSE, DodoMine, Dominik Kawula (dom.kawula), Dmytro Lytovchenko (kvakvs), Emilio Macchione (emiliomacchione), ethereal97, Fitim Qerimi (Illyrius), Fizzy_LaFizz, footbool, GiorgioHerbie, 許景翔 (gk50125012), Golden (littlegolden), Guillaume Turchini (orion78fr), Headspike, Henrik Eilmann (eilmannhenrik), henry Liu (clarinetism76), Ikki_di_Phoenix, imacsergey, ipd, jesuslobo, JJTspanishguy, John Lok Ho (lok91101), jrthsr700tmax, Kevin (kevinlin18), Kevin Eichenberg (BanditBloodwyn), kianzarrin (kian.zarrin), Kuba Kukuč Pátek (jakubpatek), krzychu124 (krzychu1245), Lasm Gratel (LasmGratel), lennier3 (lennier), LordOrodreth, Mahmoud Al Gammal (algammal.design), MarkTrefi, mashitaro, Mbyron26, Mehmet Ömer Tüzer (PELLQ), mjm92150, MrLojnat, neinnew, Nithox, obv, Ömer Kılıç (TonsilStone), perry7484, PhoneixS, Rıdvan SAYLAR (ridvan.saylar), Rodirik, Ryan Gleeson (Ryan_7149), Sergio Martins (sergio.paulo.martins), shg166 (shg166_csl), Simone Delvecchio (DelvecchioSimone), siSINka, smaux95, Stuart, Tayfun [Typhoon] (Koopr), 田七不甜 (TianQiBuTian), TwotoolusFLY_LSh.st, StummeH, tmechen, Veli Taikan (velitaikan279), vicugna-pacos, vitalii201, Vladimír Krajč (mkkyy), Warintorn Choorasri (warintorn.choorasri), wxf26054 (Alex-Yang), しょしょ02 (yamadatarounohosi)!

**Comment by krzychu124** [comment]
Hehe, a lot of names 😄 We may hit character count limit on the workshop page 😕 
Maybe we should all small modal activated with a button somewhere on the options page, or just crowdin button to jump to translation, similar to what IMT has?

I was thinking about borrowing "What's new" panel from IMT. Might be useful for 11.6 since we've made a lot of changes, and sometimes users are not aware of fixes/changes 🤔 Maybe even extend it with additional action button when used for notifying that some action is required in case of issues or changes in the code, like that one some time ago with parkingAI loop bug - required vehicles clearing to fix it

**Reply by originalfoo** [reply]
Yup, big list. Took ages to compile because as far as I can tell Crowdin doesn't have a proper list of contributors to a project (I found some lists in crowdin UI, but it hardly listed any users who have participated). We can maybe list in the release announcement for 11.6 or use separate 'discssion; tab page in steam workshop?

> crowdin button to jump to translation, similar to what IMT has?

Yup, that would be good.

> I was thinking about borrowing "What's new" panel from IMT

Yes, definitely! Related: #118

> Maybe even extend it with additional action button when used for notifying that some action is required in case of issues or changes in the code

Yup; IMO any change to pathfinder should require users to "clear traffic" after an update - but the thing there it's _per city_ not just _per mod update_. Related: #485 

Something akin to "What's new" panel could possibly also be used to overcome issue discussed in #1240 - ie. an alternate to the game's inbuilt tutorial message system (although for Lane Arrows tool, IMO just changing the button icons - eg. from lane arrow to lane connector, to let user know lane connectors used in that lane - would be a better place to start).
